### PR TITLE
Typos: `benchmark` folder

### PR DESCRIPTION
**Pull request by Sajjon**
Fix typos in `Benchmark` folder.

This is one batch of many PRs fixing typos, [see the tracking issue.](https://github.com/swiftlang/swift/issues/75022)

### CODE REVIEW

File: benchmark/single-source/SortIntPyramids.swift [hunk 1]
```diff
@@ -37,7 +37,7 @@ let pyramidTemplate: [Int] = (1...pH) + (1...pH).reversed()
 // A^R - reversed array A, + - array concatenation operator,
 // A indices are in range 1...A.length.
 // define adjacent pyramid as A + A^R + A + A^R,
-// defne adjacent pyramid hight as A[A.length].
+// defne adjacent pyramid height as A[A.length].
```
Comment (AnthonyLatsis) [inline benchmark/single-source/SortIntPyramids.swift:40]: "defne" is also a typo.

**Review by AnthonyLatsis — APPROVED** [review]

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

File: benchmark/single-source/SortIntPyramids.swift (hunk 1 above)
Comment (Sajjon) [reply benchmark/single-source/SortIntPyramids.swift]: Thanks, fixed!

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Review by eeckstein — APPROVED** [review]

**Pull request merged**